modules/menu/menu.c

```diff
@@ -402,6 +402,49 @@ static int cmd_ua_next(struct re_printf *pf, void *unused)
 	return err;
 }
 
+static int cmd_ua_find(struct re_printf *pf, void *arg)
+{
+	const struct cmd_arg *carg = arg;
+	struct ua *ua = NULL;
+	int err;
+
+	(void)pf;
+
+	if (str_isset(carg->prm)) {
+
+		mbuf_rewind(menu.dialbuf);
+		(void)mbuf_write_str(menu.dialbuf, carg->prm);
+
+		ua = uag_find_aor(carg->prm);
+	}
+	else if (menu.dialbuf->end > 0) {
+
+		char *aor;
+
+		menu.dialbuf->pos = 0;
+		err = mbuf_strdup(menu.dialbuf, &aor, menu.dialbuf->end);
+		if (err)
+			return err;
+
+		ua = uag_find_aor(aor);
+
+		mem_deref(aor);
+	}
+
+	if (!ua) {
+		warning("menu: ua_find failed\n");
```

Comment:
perhaps also print the string `carg->prm` so it becomes a bit more user friendly :)